src/commands/configure.ts

```diff
@@ -44,28 +46,44 @@ export default class Configure extends Command {
       return;
     }
 
-    console.log(
-      chalk.yellow(
-        "This process will make changes to the file system of your project."
-      )
-    );
-    console.log(
-      chalk.yellow(
-        "It is highly recommended that you do this after committing or stashing all working changes so you can revert with git if needed."
-      )
+    const gitCheck = new Listr(
+      [
+        {
+          title: "Checking Git status...",
+          task: (ctx, task) =>
+            execAsync(["git", "status", "--porcelain"].join(" "), {
```

Comment:
Listr tasks can be async. You don't need to use `.then` chaining you can just do async/await

Reply:
Actually, these changes shouldn't be part of this PR it looks like your two separate features are both present in both PRs